packages/scripts/src/lib/plugin/handlers/fields/calculations.ts

```diff
@@ -0,0 +1,112 @@
+import type Freeform from '@components/front-end/plugin/freeform';
+// @ts-ignore
+import ExpressionLanguage from 'expression-language';
+import type { FreeformHandler } from 'types/form';
+
+class Calculation implements FreeformHandler {
+  freeform: Freeform;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  expressionLanguage: any;
+
+  constructor(freeform: Freeform) {
+    this.freeform = freeform;
+    this.expressionLanguage = new ExpressionLanguage();
+
+    this.reload();
+  }
+
+  valueOrdination = (value: string): string | number | boolean => {
+    const lowercasedValue = value.toLowerCase();
+
+    if (lowercasedValue === 'true') {
+      return true;
+    } else if (lowercasedValue === 'false') {
+      return false;
+    }
+
+    return isNaN(Number(value)) ? value : Number(value);
+  };
+
+  reload = () => {
+    const pickers = this.freeform.form.querySelectorAll('input[type="calculation"]');
```

Comment:
There are no valid HTML input types called `calculation`. I'm not sure this is a correct approach. The type should be `text` for the input itself.

I think you can select the pickers by the attribute.

```suggestion
    const pickers = this.freeform.form.querySelectorAll('input[data-calculations]');
```